contracts/deployables/erc20/VotesERC20LockableV1.sol

```diff
@@ -59,9 +59,6 @@ contract VotesERC20LockableV1 is IVotesERC20LockableV1, VotesERC20V1 {
     }
 
     function lock(bool locked_) external virtual override onlyOwner {
-        if (locked_ == _locked) {
-            revert CannotSwitchLockState(locked_);
-        }
```

Comment:
this is another I would consider leaving in - since we know for sure it was a mistake if they pass in a new lock state that's the same as the current lock state

Reply:
yeah, but to argue against that, we don't do any other checks like this when doing setters for things like updating uints.

Reply:
i'd prefer to leave this as is